internal/github/languages.go

```diff
@@ -0,0 +1,112 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+)
+
+type Language struct {
+	Size  int
+	Name  string
+	Color string
+}
+
+type Languages []Language
+
+func (l Languages) String() string {
+	str := ""
+
+	for _, lang := range l {
+		str += fmt.Sprintf("**%s** %d bytes\n", lang.Name, lang.Size)
```

Comment:
can't use the language hexcode color in formatting since it doesn't seem like discord allows text to be individually colored via a hexcode.